src/mon/MDSMonitor.cc

```diff
@@ -669,6 +669,13 @@ bool MDSMonitor::prepare_beacon(MonOpRequestRef op)
           pending_daemon_health_rm.insert(erased_gid);
         }
       }
+
+      auto fscid = pending_fsmap.mds_roles.at(gid);
+      auto fs = pending_fsmap.get_filesystem(fscid);
+      mon->clog->info() << "MDS daemon " << m->get_name() << " finished "
```

Comment:
Do we really want to avoid the "mds.foo" daemon names? They're a bit of a cephism but they're precise...